actions/actions.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/local/sbin/charm-env python3
+import os
+import pwd
+import grp
+import sys
+import tarfile
+import shutil
+
+from datetime import datetime
+from typing import List
```

Comment:
Avoid the use of `typing` (supported from `py35+`) because the layer is supported in `trusty` (`py34`),